certification/internal/engine/engine.go

```diff
@@ -93,6 +99,19 @@ func (c *CraneEngine) ExecuteChecks() error {
 		return fmt.Errorf("%w: %s", errors.ErrExtractingTarball, err)
 	}
 
+	// if --submit flag is set write json files to disk to be used to send to pyxis after all checks are complete
+	if viper.GetBool("submit") {
+		err := writeCertImage(img)
+		if err != nil {
+			return err
+		}
+
+		err = writeRPMManifest(containerFSPath)
+		if err != nil {
+			return err
+		}
+	}
```

Comment:
I feel this block should come after the check execution. I am unsure of the use case: can a partner submit the preflight result even though some checks are failing?